integration_tests/client/datasets/test_groundtruth.py

```diff
@@ -197,20 +197,18 @@ def test_add_groundtruth(
     with pytest.raises(TypeError):
         dataset.add_groundtruth("not_a_gt")  # type: ignore
 
-    # make sure we get a warning when passing a ground truth without annotations
-    with pytest.warns(UserWarning):
-        dataset.add_groundtruth(
-            GroundTruth(
-                datum=Datum(
-                    uid="uid",
-                    metadata={
-                        "height": 200,
-                        "width": 150,
-                    },
-                ),
-                annotations=[],
-            )
+    dataset.add_groundtruth(
```

Comment:
whats the point of this (and similary the empty `add_prediction` below) now? is it to verify nothing goes wrong when adding an empty groundtruth or prediction?

Reply:
if we want to check that no warnings are thrown we can do

```
with warnings.catch_warnings():
    warnings.simplefilter("error")
    dset.add_groundtruth(...)
```

(see second bullet point here https://docs.pytest.org/en/7.1.x/how-to/capture-warnings.html#additional-use-cases-of-warnings-in-tests)